static/src/javascripts/projects/commercial/modules/prebid/bid-config.js

```diff
@@ -349,12 +349,12 @@ const isPbTestOn = (): boolean => !isEmpty(pbTestNameMap());
 const inPbTestOr = (liveClause: boolean): boolean => isPbTestOn() || liveClause;
 
 /* Bidders */
-const appNexusBidder: PrebidBidder = {
-    name: 'and',
+const appNexusBidder = (name: string): PrebidBidder => ({
```

Comment:
Suggestion, since the bid params are going to rely on `isInAuRegion()` too:

```js
const appNexusBidder = (isInAuRegion: boolean): PrebidBidder => ({
   name: isInAuRegion() ? 'and' : 'and-uk-row',
```

Then we can pass the bool down to getAppNexusBidParams so it can get involved in our fuuuuun if else decision tree that I will shortly contribute to 😅 

e.g. `getAppNexusBidParams(sizes, isInAuRegion)`

Reply:
Ah yes, if we do it this way with one adapter then let's pass the boolean!